sandbox-1.7 traces static apps so disable the qa notice as it just scares
users ... dont want scary stuff in stable!

--- libsandbox/wrapper-funcs/__wrapper_exec.c
+++ libsandbox/wrapper-funcs/__wrapper_exec.c
@@ -221,7 +221,6 @@
 		if (!FUNCTION_SANDBOX_SAFE(path))
 			return result;
 
-		sb_check_exec(path, argv);
 	}
 #endif
 
